src/icons/Contract/Contract.native.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
this should be collapse right? Expand, collapse

Reply:
I just went with Expand's antonym Contract 😛 Was confused in choosing between shrink, condense, contract. Collapse, not sure if it appropriate.

Reply:
Expand <> Collapse sounds better.